Sources/Sentry/Public/SentryHttpStatusCodeRange.h

```diff
@@ -0,0 +1,27 @@
+#import "SentryDefines.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * The Http status code range. Example for a range: 400 to 499, 500 to 599, 400 to 599 The range is
+ * inclusive so the min and max is considered part of the range.
+ *
+ * Example for a single status code 400, 500
+ */
+NS_SWIFT_NAME(HttpStatusCodeRange)
+@interface SentryHttpStatusCodeRange : NSObject
+SENTRY_NO_INIT
+
+@property (nonatomic, readonly) NSInteger min;
+
+@property (nonatomic, readonly) NSInteger max;
+
+- (instancetype)initWithMin:(NSInteger)min max:(NSInteger)max;
+
+- (instancetype)initWithStatusCode:(NSInteger)statusCode;
+
+- (BOOL)isInRange:(NSInteger)statusCode;
```

Comment:
`m`: This doesn't need to be public. We could create a private category only to expose it in the SDK. An example for that https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/include/SentryScope%2BPrivate.h

Reply:
I'm a bit confused with this example, the interface is called `SentryScope` but the class `SentryAttachment`
The properties `not` only related to attachments, I also didn't find the `.m` of this class.